### PR TITLE
Module loading priorities

### PR DESCRIPTION
**Pull request by Heufneutje**
Works consistently on the testnet. I went ahead and gave APIKeys and the command handler a higher priority. The command handler doesn't necessarily need it but it seemed like a good thing to do.

### CODE REVIEW

**Review by HubbeKing — COMMENTED** [review]
Looks good to me, could perhaps use some documentation on how the loadingPriority works, though?
I'm assuming 10 is higher priority than 1, but a docstring or comment clarifying which order modules are sorted when loading would be useful, I think.

**Comment by Heufneutje** [comment]
Probably yes.

**Comment by Heufneutje** [comment]
Done.

**Review by HubbeKing — APPROVED** [review]
Looks good to me then!

File: desertbot/moduleinterface.py [hunk 1]
```diff
@@ -63,6 +63,11 @@ class BotModule(object):
     def __init__(self):
         self.logger = logging.getLogger('desertbot.{}'.format(self.__class__.__name__))
 
+        self.loadingPriority = 1
+        """
+        Increase this number in the module's class it the module should be loaded before other modules.
```
Comment (StarlitGhost) [inline desertbot/moduleinterface.py]: *if

**Comment by StarlitGhost** [comment]
Looks good to me aside from that one typo :)

**Comment by Heufneutje** [comment]
Fixed the typo. I'll go ahead and merge this.

**Pull request merged**